trio/_core/_run.py

```diff
@@ -1376,7 +1377,13 @@ def run_impl(runner, async_fn, args):
             except StopIteration as stop_iteration:
                 final_result = Value(stop_iteration.value)
             except BaseException as task_exc:
-                final_result = Error(task_exc)
+                # Store for later, removing uninteresting top frames:
+                #   1. trio._core._run.run_impl()
+                #   2. contextvars.Context.run() (< Python 3.7 only)
+                tb_next = task_exc.__traceback__.tb_next
+                if sys.version_info < (3, 7):  # pragma: no cover
```

Comment:
`pragma: no cover` is for cases where we don't even want to have test coverage. Here, not only is it useful to have tests that cover both branches of the `if`, we actually have those tests :-). (We test on multiple python versions and combine the coverage.) So we shouldn't use `pragma: no cover` here.

Reply:
Oh that's good--  I assumed the coverage testing was looking at each version run individually rather than the aggregate.